internal/daysteps/daysteps.go

```diff
@@ -13,8 +20,48 @@ const (
 
 func parsePackage(data string) (int, time.Duration, error) {
 	// TODO: реализовать функцию
+	if data == "" {
+		return 0, 0, errors.New("no data to process")
+	}
+	slData := strings.Split(data, ",") // конвертация строки в слайс slDate
+	if len(slData) != 2 {
+		return 0, 0, errors.New("incorrect amount of data")
+	}
+
+	steps, err := strconv.Atoi(slData[0]) // преобразование в int
+	if err != nil {
+		return 0, 0, errors.New("step count conversion error")
```

Comment:
Теряешь сообщение об ошибки
```suggestion
		return 0, 0, fmt.Errorf("conversion error: %w", err)
```